app/src/main/res/layout/fragment_admin_hub.xml

```diff
@@ -21,6 +21,8 @@
             android:layout_height="wrap_content"
             android:gravity="center|fill_vertical"
             android:orientation="horizontal"
+            android:layout_marginTop="50dp"
+            android:layout_marginBottom="10dp"
```

Comment:
If it works great, but we should be careful when we add this kind of constraints as the emulator used by Cirrus is much smaller than the one we use on our computer, this can lead to unexepcted error/superposition.

Reply:
I'm not sure I edited this file. Must have been when I merged with master on Florian's branch.